client/state/themes/selectors.js

```diff
@@ -422,7 +422,7 @@ export function getThemePurchaseUrl( state, themeId, siteId ) {
 		return null;
 	}
 	if ( config.isEnabled( 'jetpack/pijp' ) && isJetpackSite( state, siteId ) ) {
-		return `/plans/${ getSiteSlug( state, siteId ) }`;
+		return getPlanUpgradeUrl( state, siteId );
```

Comment:
Oh, I meant to not call that other selector from here, but directly from `planUpgrade`'s `getUrl` in `theme-options` :slightly_smiling_face: